client/src/Pages/Footer.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
+import "../Styles/Footer.css";
+
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="row mb-0 mt-0">
+        <div className="mb-0 mt-0 col-12 col-md-4 links">
+          <h4>Useful Links</h4>
+          <ul>
+            <li>
+              <a href="#"> Dashboard</a>{" "}
+            </li>
+            <li>
+              {" "}
+              <a href="#"> About</a>{" "}
+            </li>
+            <li>
+              {" "}
+              <a href="#"> Notes</a>{" "}
+            </li>
+            <li>
+              {" "}
+              <a href="#"> Connect</a>{" "}
```

Comment:
We can have a Profile Section as well. It would be the page where the User Profile Page would be rendered altogether.